### ISSUE TITLE
Repo does not run properly

### ISSUE THREAD

**Issue by paralin** [Urigo/meteor-angular2.0-socially#86]
Cloning the repo and doing npm install and then meteor:

```
=> Started proxy.                             
=> Started MongoDB.                           
client/imports/auth/recover.ts (1, 25): Cannot find module 'angular2/core'.
client/imports/auth/recover.ts (3, 22): Cannot find module 'angular2/router'.
client/imports/auth/recover.ts (4, 53): Cannot find module 'angular2/common'.
client/imports/auth/recover.ts (6, 26): Cannot find module 'angular2/router'.
client/imports/auth/recover.ts (31, 7): Cannot find name 'Accounts'.
client/imports/auth/signup.ts (1, 25): Cannot find module 'angular2/core'.
client/imports/auth/signup.ts (3, 22): Cannot find module 'angular2/router'.
client/imports/auth/signup.ts (4, 53): Cannot find module 'angular2/common'.
client/imports/auth/signup.ts (6, 26): Cannot find module 'angular2/router'.
client/imports/auth/signup.ts (32, 7): Cannot find name 'Accounts'.
client/imports/parties-list/parties-list.ts (23, 2): Supplied parameters do not match any signature of call target.
client/imports/party-details/party-details.ts (19, 2): Supplied parameters do not match any signature of call target.
client/app.ts (16, 26): Cannot find module 'angular2/router'.
client/app.ts (21, 60): Cannot find module 'meteor/socially-browser'.
client/app.ts (22, 74): Cannot find module 'meteor/socially-mobile'.
client/app.ts (37, 2): Supplied parameters do not match any signature of call target.
client/app.ts (39, 9): Cannot find namespace 'Meteor'.
client/app.ts (47, 7): Cannot find name 'Meteor'.
Compiling main app HTML file: client/index.html
Compiling HTML template: auth/login.html
Compiling HTML template: app/app.html
Compiling HTML template: app/app.html
Compiling HTML template: auth/login.html
Compiling HTML template: client/imports/auth/recover.html
Compiling HTML template: client/imports/auth/signup.html
Compiling HTML template: client/imports/parties-form/parties-form.html
Compiling HTML template: client/imports/parties-list/parties-list.html
Compiling HTML template: client/imports/party-details/party-details.html
```

**Comment by Urigo** [comment]
yes the last chapters (18 and on) are not fully updated yet but until 17 it should work

**Comment by yvesbouvier** [comment]
Hi Uri,

I cloned the project, and use:
`git reset --hard 35bcbc950404b4404de7ef50d05fa3e3a08c5604` (hash for step 17)

That still gives me errors, was I suppose to do it differently ? Thanks for the help

**Comment by kamilkisiela** [comment]
@yvesbouvier I updated Socially 2.0 recently. Could you try to check it again with the latest version?

You can use `step_XX` tag since they have been updated too.

I checked every step to see if there are any warnings so you not suppose to see them anymore :+1:

**Comment by omararturo** [comment]
Hi, Its work for me
**package.json**

```
{
  "name": "socially",
  "private": true,
  "scripts": {
    "start": "meteor run"
  },
  "dependencies": {
    "@angular/common": "2.0.0-rc.1",
    "@angular/compiler": "2.0.0-rc.1",
    "@angular/core": "2.0.0-rc.1",
    "@angular/platform-browser": "2.0.0-rc.1",
    "@angular/platform-browser-dynamic": "2.0.0-rc.1",
    "@angular/router-deprecated": "^2.0.0-rc.1",
    "@angular2-material/core": "^2.0.0-alpha.4",
    "@angular2-material/checkbox": "^2.0.0-alpha.4",
    "@angular2-material/input": "^2.0.0-alpha.4",
    "@angular2-material/toolbar": "^2.0.0-alpha.4",
    "angular2-google-maps": "0.10.0",
    "angular2-meteor": "0.5.5",
    "angular2-meteor-accounts-ui": "0.4.3",
    "angular2-meteor-auto-bootstrap": "0.5.5",
    "angular2-pagination": "0.2.1",
    "es6-shim": "0.35.1",
    "meteor-node-stubs": "^0.2.3",
    "ng2-material": "0.4.1",
    "reflect-metadata": "0.1.2",
    "rxjs": "5.0.0-beta.6",
    "zone.js": "0.6.12"
  }
}
```

First use:
 **npm isntall**

then add in **app.ts** and **login.ts** this:
`import {Meteor} from 'meteor/meteor';`

then add in **recover.ts** and **singup.ts** this:
`import {Accounts} from 'meteor/accounts-base';`

then change **@InjectUser()** in **app.ts**, **parties-list.ts** and **party-details.ts** for
`@InjectUser('')`

and Enjoy!!!!

**Comment by kamilkisiela** [comment]
@omararturo Exactly! I updated the tutorial earlier today in the same way.

About `@InjectUser('')`. If Urigo/angular2-meteor-accounts-ui#15 will be merged and published then the argument of InjectUser will be optional (so `@InjectUser()`).

**Comment by Urigo** [comment]
A new version has been published, @paralin can you check if @kamilkisiela fix fixed it for you?

**Comment by haluvibe** [comment]
modules-runtime.js:25Uncaught Error: Cannot find module 'angular2-meteor-polyfills/browser'
Im on step 0.8

**Comment by haluvibe** [comment]
reflective_injector.js:679Uncaught EXCEPTION: Error during instantiation of NgZone! (ApplicationRef -> ApplicationRef_ -> NgZone).

**Comment by haluvibe** [comment]
And if I clone the whole repo I get: install.js:78Uncaught Error: Cannot find module '@angular/router'

**Comment by haluvibe** [comment]
just tried again on a different computer - NPM V3 installed latest and nodejs latest installed.

Getting these warnings and errors when npm install is run:
npm WARN ng2-material@0.4.1 requires a peer of @angular/router@2.0.0-rc.1 but none was installed.
npm WARN ng2-material@0.4.1 requires a peer of @angular2-material/progress-bar@^2.0.0-alpha.4 but none was installed.
npm WARN ng2-material@0.4.1 requires a peer of @angular2-material/progress-circle@^2.0.0-alpha.4 but none was installed.
npm WARN ng2-material@0.4.1 requires a peer of @angular2-material/radio@^2.0.0-alpha.4 but none was installed.
npm WARN ng2-material@0.4.1 requires a peer of @angular2-material/sidenav@^2.0.0-alpha.4 but none was installed.
npm WARN ionic-angular@2.0.0-beta.8 requires a peer of @angular/http@^2.0.0-rc.1 but none was installed.
npm WARN ionic-angular@2.0.0-beta.8 requires a peer of @angular/router@^2.0.0-rc.1 but none was installed.
├── UNMET PEER DEPENDENCY @angular/http@^2.0.0-rc.1
├── UNMET PEER DEPENDENCY @angular/router@^2.0.0-rc.1
├── UNMET PEER DEPENDENCY @angular2-material/progress-bar@^2.0.0-alpha.4
├── UNMET PEER DEPENDENCY @angular2-material/progress-circle@^2.0.0-alpha.4
├── UNMET PEER DEPENDENCY @angular2-material/radio@^2.0.0-alpha.4
├── UNMET PEER DEPENDENCY @angular2-material/sidenav@^2.0.0-alpha.4

Then when I run meteor I get these errors:

Unable to resolve some modules:

  "@angular/router" in /Users/paulhayes/Sites/meteor-angular2.0-socially-master/node_modules/ionic-angular/config/bootstrap.js (web.browser)
  "@angular/http" in /Users/paulhayes/Sites/meteor-angular2.0-socially-master/node_modules/ionic-angular/config/bootstrap.js (web.browser)

And I get a blank page with a ton of console errors caused clearly by the above when I load it in a web page.

Also, not even the first page of the tutorial works for me.

**Comment by jrfoxw** [comment]
I'm getting the same thing lesson 1-3. Tried it on virtual and actual servers (Ubunto, centos) and getting the same errors.

![meteor_angular2_errors1](https://cloud.githubusercontent.com/assets/13408697/16431188/1ad59132-3d45-11e6-87fb-e273b781f4eb.PNG)
![meteor_angular2_errors2](https://cloud.githubusercontent.com/assets/13408697/16431189/1ad6d04c-3d45-11e6-8a66-58d01dfe8fab.PNG)

**Comment by kamilkisiela** [comment]
@jrfoxw 

You should use:

``` bash
meteor npm install angular2-meteor-polyfills --save
```

@haluvibe Those issues has been resolved with the latest version of Socially

**Comment by kamilkisiela** [comment]
@mention me if this is still an issue.

**Comment by numerized** [comment]
@kamilkisiela Impossible from the last repo to install socially from your last package.json

{
  "name": "socially",
  "private": true,
  "scripts": {
    "start": "meteor run"
  },
  "dependencies": {
    "@angular/common": "^2.0.0-rc.4",
    "@angular/compiler": "^2.0.0-rc.4",
    "@angular/core": "^2.0.0-rc.4",
    "@angular/http": "^2.0.0-rc.4",
    "@angular/platform-browser": "^2.0.0-rc.4",
    "@angular/platform-browser-dynamic": "^2.0.0-rc.4",
    "@angular/router": "3.0.0-beta.2",
    "@angular2-material/core": "2.0.0-alpha.4",
    "@angular2-material/checkbox": "2.0.0-alpha.4",
    "@angular2-material/input": "2.0.0-alpha.4",
    "@angular2-material/toolbar": "2.0.0-alpha.4",
    "angular2-file-drop": "^0.0.3",
    "angular2-google-maps": "0.12.0",
    "angular2-meteor": "^0.5.6",
    "angular2-meteor-accounts-ui": "0.5.0",
    "angular2-meteor-auto-bootstrap": "^0.5.6",
    "angular2-meteor-polyfills": "^0.1.1",
    "angular2-pagination": "0.2.1",
    "es6-shim": "^0.35.1",
    "gm": "^1.22.0",
    "ionic-angular": "^2.0.0-beta.9",
    "ionic-selector": "0.0.2",
    "meteor-node-stubs": "^0.2.3",
    "ng2-material": "0.4.1",
    "reflect-metadata": "^0.1.2",
    "rxjs": "5.0.0-beta.6",
    "zone.js": "^0.6.12"
  }
}

I've tried all manually untill step pagination which is not working neither, if I bypass it, the angular2-google-maps is not installing neither because of a UNMET PEER DEPENDENCY.

node -v
v6.2.2

npm -v
3.10.5

Thanks, If you need more infos just let me know.

**Comment by helloncanella** [comment]
Same.

**Comment by Urigo** [comment]
we just deployed a new version.
If it still doesn't work for you please open a new issue

**Comment by tafelito** [comment]
Hi All, I'm not sure if its realted to this issue, but I'm following the tutorial and I having this issue too, 

```
Unable to resolve some modules:

  "@angular/http" in /Users/xxx/Documents/socially/node_modules/@angular/material/material.umd.js (web.browser
)
```

Could that be related to this issue?

Cheers

**Comment by dotansimha** [comment]
@tafelito, please open new issues and don't comment in old & closed issues because it's hard to keep track on them. 

Regarding you issue, did you tried:

``` js
npm install @angular/http
```

?

**Comment by tafelito** [comment]
@dotansimha ok thanks, I will open a new issue next time! I already fixed it, I added the http module, but I had to force the version 2.1.1 to be aligned to the rest of the plugins. When I added that individually it wasn't working, I think it was giving the same error I put on issue #145. To solved it I removed all node modules and do `meteor npm instal`l again, with the updated packages.json and after that, all packages were in the correct version
